3_javascript/javascript2/week1/scripts.js

```diff
@@ -0,0 +1,67 @@
+/// 1.2 Find and count the Danish letters ///
+
+function countDanishLetters(inputString) {
+  const danishLetters = {
+    æ: 0,
+    ø: 0,
+    å: 0,
+  };
+
+  for (const char of inputString) {
+    if (danishLetters[char]) {
+      danishLetters[char]++;
+    }
+  }
+  const total = Object.values(danishLetters).reduce(
+    (acc, count) => acc + count,
+    0
+  );
+  return { total, danishLetters };
+}
+
+/// Examples of what it returns ///
+const danishString = 'Jeg har en blå bil';
+countDanishLetters(danishString); // returns {total: 1, å: 1}
+
+const danishString2 = 'Blå grød med røde bær';
```

Comment:
Make sure to console log the results to check if they are correct. I got `{ total: 0, danishLetters: { 'æ': 0, 'ø': 0, 'å': 0 }`for both of them